--- zstd-0.5.1/lib/Makefile.orig	2016-04-09 17:48:36.367934319 +0200
+++ zstd-0.5.1/lib/Makefile	2016-04-09 17:56:18.837914911 +0200
@@ -79,9 +79,9 @@
 
 .PHONY: default all clean install uninstall
 
-default: clean libzstd
+default: libzstd
 
-all: clean libzstd
+all: libzstd
 
 libzstd: $(ZSTD_FILES)
 	@echo compiling static library
@@ -110,7 +110,7 @@
              -e 's|@VERSION@|$(VERSION)|' \
              $< >$@
 
-install: libzstd libzstd.pc
+install:  libzstd.pc
 	install -d -m 755 $(DESTDIR)$(LIBDIR)/pkgconfig/ $(DESTDIR)$(INCLUDEDIR)/
 	install -m 755 libzstd.$(SHARED_EXT_VER) $(DESTDIR)$(LIBDIR)/libzstd.$(SHARED_EXT_VER)
 	cp -a libzstd.$(SHARED_EXT_MAJOR) $(DESTDIR)$(LIBDIR)
