--- zstd-1.1.2/lib/Makefile.orig	2016-12-26 12:28:15.143603624 +0100
+++ zstd-1.1.2/lib/Makefile	2016-12-26 12:30:33.816935375 +0100
@@ -105,7 +105,7 @@
              -e 's|@VERSION@|$(VERSION)|' \
              $< >$@
 
-install: libzstd.a libzstd libzstd.pc
+install: libzstd.a $(LIBZSTD) libzstd.pc
 	install -d -m 755 $(DESTDIR)$(LIBDIR)/pkgconfig/ $(DESTDIR)$(INCLUDEDIR)/
 	install -m 755 libzstd.$(SHARED_EXT_VER) $(DESTDIR)$(LIBDIR)
 	cp -a libzstd.$(SHARED_EXT_MAJOR) $(DESTDIR)$(LIBDIR)
